string ssid
float32 quality
int32 signal
int32 noisy
float32 freq
int32 channel